trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/check-the-information/index.test.ts

```diff
@@ -22,23 +31,44 @@ describe('controllers/utilisation-reports/record-corrections/check-the-informati
         params: { reportId, feeRecordId: feeRecordId.toString() },
       });
 
+      const bank = { name: 'Test bank', id: '129' };
+      const reportPeriod = { start: { month: 7, year: 2024 }, end: { month: 7, year: 2024 } };
+      const correctionRequestDetails = {
+        facilityId: '0012345678',
+        exporter: 'Test company',
+        reasons: [RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT, RECORD_CORRECTION_REASON.OTHER],
+        additionalInfo: 'The facility ID does not match the facility ID held on file',
+        contactEmailAddresses: ['one@email.com', 'two@email.com'],
+      };
+
+      const mockApiResponse = {
+        bank,
+        reportPeriod,
+        correctionRequestDetails,
+      };
+
+      jest.mocked(api.getFeeRecordCorrectionRequestReview).mockResolvedValue(mockApiResponse);
+
       // Act
-      getRecordCorrectionRequestInformation(req, res);
+      await getRecordCorrectionRequestInformation(req, res);
 
       // Assert
+      const expectedReasons = mapReasonsToDisplayValues(correctionRequestDetails.reasons).join(', ');
```

Comment:
How would you feel about pulling these expectation variables up into the arrange section?